src/pages/signUp.module.css

```diff
@@ -1,12 +1,18 @@
-.signUp {
+@import url("../styles/media.css");
+
+.root {
   display: flex;
   flex-direction: row-reverse;
+
+  @media (--media-max-lg) {
+    display: none;
+  }
 }
 
 .signUp-card {
```

Comment:
setup loader for BEM in vite.
https://medium.com/trabe/using-bem-conventions-in-css-modules-leveraging-custom-webpack-loaders-fd985f72bcb2